.bowerrc

```diff
@@ -0,0 +1,3 @@
+{
+    "directory": "web/assets/vendor"
```

Comment:
Agree!. Bower files must be on the root and only compiled/processed files inside web. This files should be compiled/processed using asetic or grunt